src/html/mod.rs

```diff
@@ -92,6 +92,23 @@ impl Html {
         }
     }
 
+    /// Fast select open edges that match selector into vector of elements
+    pub fn matches<'a, 'b>(&'a self, selector: &'b Selector) -> Vec<ElementRef> {
```

Comment:
Why always return a `Vec` instead of something like

```
pub fn matches<'a>(&'a self, selector: &'a Selector) -> impl Iterator<Item = ElementRef> + 'a
```

?

Reply:
No reason, updated thanks!